docs/components/similaritygraph.rst

```diff
@@ -24,7 +24,7 @@ Example
 
 .. raw:: html
 
-    <svg id="similaritygraph-example" width="700" height="700"></svg>
+    <div id="similaritygraph-example" width="700" height="700"></svg>
```

Comment:
Oops! I guess I'll have to blag about this :-P

See 3d13295a8431c400525de305a9f5c68b77606c49.